packages/compass/src/main/menu.ts

```diff
@@ -434,7 +434,7 @@ class CompassMenu {
     debug(`WINDOW ${bw.id} load()`);
 
     if (bw.id !== this.currentWindowMenuLoaded) {
-      if (this.windowState.has(bw.id)) {
+      if (!this.windowState.has(bw.id)) {
```

Comment:
Here's the fix, looks like this if was flipped in the update to ts https://github.com/mongodb-js/compass/pull/2493/files#diff-b923c6a83071ad3fc92921ec6d13324528e3fb1972045636ae59f4b7f0bf4e9dL459

Reply:
Woah, good catch!